templates/deno/package.json

```diff
@@ -1,15 +1,17 @@
 {
   "name": "remix-template-deno",
   "private": true,
+  "description": "",
+  "license": "",
```

Comment:
Why are `description` and `license` added here?

Reply:
They aren't used at all, so they should be omitted.

Reply:
That should be done in a separate PR imo.

Let's merge these changes so all templates are consistent & we can remove it again in another PR if we want to.

Reply:
They're not used by us, but people will use them in their apps usually.

Reply:
`description`, `license` and similar fields in `package.json` are designed for providing package metadata when that package is installed as a dependency. They are used at all for apps.

Reply:
You could say the same about `private` & `sideEffects` I think?
And these are also in all `package.json`.

I'll revert these & we can discuss further in another PR to remove them in all other templates.

Reply:
`private` is used by `npm`, `yarn` and other tools so that they know this package isn't a library that will be published to NPM, so it is appropriate for apps to set `private: true`.

I think you're right about `sideEffects`. I believe that's not used if the package is actually an app.